autoload/gitmessenger/blame.vim

```diff
@@ -397,7 +397,7 @@ let s:blame.spawn_git = funcref('s:blame__spawn_git')
 
 function! s:blame__start() dict abort
     call self.spawn_git(
-        \ ['--no-pager', 'blame', self.blame_file, '-L', self.line . ',+1', '--porcelain'],
+        \ ['--no-pager', 'blame', self.blame_file, '-L', self.line . ',+1', '--porcelain'] + split(g:git_messenger_extra_blame_args, ' '),
```

Comment:
Please also apply this change to below line.

https://github.com/rhysd/git-messenger.vim/blob/72975c9be9267689cd408a5ea6a02200502217d0/autoload/gitmessenger/blame.vim#L42

The line also runs `blame` for getting the older change. Please take care that the extra arguments must be inserted before `--` option in this case.

Reply:
no sweat @rhysd

Reply:
I've updated the code as per your request.